.travis.yml

```diff
@@ -1,13 +1,17 @@
 language: php
 
 php:
+  - 7.0
```

Comment:
was the travis.yml change discussed with core contributors?

Reply:
These changes were approved by one of the core contributors

Reply:
ok then :)